profile/README.md

```diff
@@ -0,0 +1,40 @@
+![Unleash_GitHub_Banner](https://user-images.githubusercontent.com/18158596/196736157-b20a5fbf-74ac-47cd-a991-fe6e2d2bc960.png)
+
+<p align="center">
+<b>The Open Source Feature Flagging Service</b>
+</p>
+
+![Unleash_GitHub_Mocks (2)](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)
+
+
+<!-- <img width="600" alt="Unleash Logo" src="https://user-images.githubusercontent.com/18158596/196730085-d168ff9e-8b1a-4790-8219-882336df6479.png"> -->
+
+**Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases, and, in the beginning, we were just a few contributors. Together we built a foundation for the amazing community that you see today. 🫂
```

Comment:
Ever since being told by my (Scottish) English teacher in high school that "in the beginning" is very specific to the book of Genesis and that you should always use "at the beginning" if you're not invoking religious imagery, I've tried to avoid it. I did some research now, and from what I can tell, it definitely still carries with it some of that meaning, but it's also not a 100%, so I'm not gonna be completely opposed 🤷🏼 

Also: "We wanted to fill an essential space in the development stack" Did we? I thought Ivar was just solving a need he had himself?

Regardless, I think we can re-jig these sentences some. How about this?

```suggestion
**Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases. At first, we were just a few contributors, but together we built a foundation for the amazing community that you see today. 🫂
```